tests/sync/test_chat.py

```diff
@@ -131,25 +112,24 @@ def test_return_prompt_false(self):
         prediction = co.chat("Yo what up?", return_prompt=False)
         self.assertIsInstance(prediction.reply, str)
         self.assertIsInstance(prediction.session_id, str)
-
-        with self.assertRaises(AttributeError):
-            prediction.prompt
+        assert prediction.prompt is None
 
     def test_preamble_override(self):
         preamble = "You are a dog who mostly barks"
-        prediction = co.chat("Yo what up?", preamble_override=preamble, return_prompt=True)
+        prediction = co.chat("Yo what up?", persona_prompt=preamble, return_prompt=True)
         self.assertIsInstance(prediction.reply, str)
         self.assertIsInstance(prediction.session_id, str)
         self.assertIn(preamble, prediction.prompt)
         print(prediction.prompt)
 
     def test_invalid_preamble_override(self):
-        with self.assertRaises(cohere.CohereError):
-            co.chat("Yo what up?", preamble_override=123).reply
+        with self.assertRaises(cohere.CohereError) as e:
+            co.chat("Yo what up?", persona_prompt=123).reply
+        self.assertIn('invalid type', str(e.exception)) 
 
     def test_username_override(self):
         username = "CustomUser"
-        prediction = co.chat("Yo what up?", username=username, return_chatlog=True)
+        prediction = co.chat("Yo what up?", user_name=username, return_chatlog=True)
```

Comment:
@madelinehjenkins @eddotman ?
The chat changes are simply due to the API changing under our feet and breaking tests.
On a broader note: it's not clear to me what endpoints are 'allowed' to change within the same API/SDK version and how we indicate this to users.

Reply:
As a general note, we're expecting more breaking changes with the `co.chat` endpoint over the coming month or so as it is in the experimental phase, but it will be more stable once we hit the alpha phase. So this may be a temporary situation, at least for chat.

As for this specific change from `username` to `user_name`, I don't know exactly what motivates it. My guess would be `username` feels more like a specific term for login credentials, while `user_name` more generally indicates the name of a user (more accurate in this case).

Reply:
a wait I thought this was about the main function call which was already renamed